cmd/containerd-shim/shim_freebsd.go

```diff
@@ -34,7 +35,7 @@ func setupSignals() (chan os.Signal, error) {
 	signal.Notify(signals)
 	// make sure runc is setup to use the monitor
 	// for waiting on processes
-	runc.Monitor = shim.Default
+	runc.Monitor = reaper.Default
 	return signals, nil
 }
 
```

Comment:
Not sure we want to maintain shim v1 for non-linux

Reply:
Agreed, though right now I was just trying to fix compilation errors as the existing code fails to compile.